src/core/ui.js

```diff
@@ -47,6 +47,8 @@ function ariaDecorate(elem, ariaMap) {
 
 const respecUI = hyperHTML`<div id='respec-ui' class='removeOnSave' hidden></div>`;
 const menu = hyperHTML`<ul id=respec-menu role=menu aria-labelledby='respec-pill' hidden></ul>`;
+const closeButton = hyperHTML`<button class="close-button" onclick=${() =>
```

Comment:
Nit: can we avoid creating another function?
```suggestion
const closeButton = hyperHTML`<button class="close-button" onclick=${ui.closeModal} title="Close">❌</button>`;
```

Reply:
We actually need a new function because in that case, ui will be used directly before its declaration